docs/manual-install.md

```diff
@@ -2,14 +2,14 @@
 
 1. Open Konsole and use the following commands, pressing the Enter/Return key after each:
 ```bash
-cd
+cd ~
```

Comment:
This one wasn't a typo, just a shortcut. Both versions work the same in Linux

Reply:
Correct, cd with no additional context navigates to the home directory. This is an unnecessary addition on both Linux and MacOS.

Reply:
ahh, didn't know that. Thanks!